src/l500/l500-color.cpp

```diff
@@ -105,6 +105,16 @@ namespace librealsense
                 { 2.f, "60Hz" },
                 { 3.f, "Auto" }, }));
 
+
+#ifdef __ANDROID__
+        // option to enable workaround for help weak usb hosts to support L515 devices
+        // with improved performance and stability
+        if(_fw_version >= firmware_version( "1.4.0.66" ) )
```

Comment:
This will catch FW 1.5.0.0 (the actual released FW currently) which does NOT have the changes required. Please work with FW team to get a number (1.5+) from them that you're sure will work.

Reply:
(Same in the depth sensor)

Reply:
don't know the actual fw version yet, I put in 1.5.1.0 for now, may need to do another PR to change it.